From 89a56cc0e9468857e1ce32082182680605e3a053 Mon Sep 17 00:00:00 2001
From: Dan Kegel <dank@oblong.com>
Date: Mon, 3 Feb 2014 12:27:06 -0800
Subject: [PATCH] Kludge: make -L the default as it is on Mac OS X 10.9.  We
 should remove this once we add -L to all our scripts that assume symlinks are
 followed.

---
 src/file.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/file.c b/src/file.c
index 325210e..5fd3e7e 100644
--- a/src/file.c
+++ b/src/file.c
@@ -158,7 +158,9 @@ main(int argc, char *argv[])
 		progname = argv[0];
 
 #ifdef S_IFLNK
-	flags |= getenv("POSIXLY_CORRECT") ? MAGIC_SYMLINK : 0;
+	/* flags |= getenv("POSIXLY_CORRECT") ? MAGIC_SYMLINK : 0; */
+	/* Kludge: Match OS X 10.9 behavior.  FIXME: fix scripts that assume -L is the default. */
+	flags |= MAGIC_SYMLINK;
 #endif
 	while ((c = getopt_long(argc, argv, OPTSTRING, long_options,
 	    &longindex)) != -1)
-- 
1.8.3.4 (Apple Git-47)

